vip-helpers/vip-utils.php

```diff
@@ -1640,3 +1640,19 @@ function vip_get_hyper_servers( $hyperdb, $operation = 'all', $dataset = 'global
 
 	return $hyperdb->hyper_servers[ $dataset ];
 }
+
+/**
+ * Checks if a given domain is a VIP convenience domain. Examples of a VIP convenience
+ * domain are `example.go-vip.co` and `example.go-vip.net`.
+ *
+ * @param string $domain Domain to check
+ * @return bool True if the domain is a convenience domain
+ */
+function is_vip_convenience_domain( string $domain ) {
```

Comment:
Slightly surprised that these utility functions aren't already under a namespace. There's a tiny but non-zero chance of a namespace clash here.

Reply:
I did an audit already and there shouldn't be any clashes.